tests/python/relax/test_transform_legalize_ops_manipulate.py

```diff
@@ -1080,7 +1081,8 @@ def main(x: R.Tensor(("a", "b", "c"), "float32")) -> R.Tensor(("b", 1), "float32
     # fmt: on
 
     mod = LegalizeOps()(CollapseSumTo)
-    tvm.ir.assert_structural_equal(mod, Expected)
+    # TODO(@relax-team): Uncomment when this is supported
+    # tvm.ir.assert_structural_equal(mod, Expected)
```

Comment:
These were giving false negatives for my IDE due to having an undefined variable. We probably should not keep tests that don't work

Reply:
+1 remove the tests that don't work, not difficult to get it back from the commit history if would like to enable them

Reply:
Removed. If others object, it's easy to restore them.